lib/Installation/DiskActivation/DASDDiskManagementPage.pm

```diff
@@ -0,0 +1,60 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Handles DASD Disk Management page
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+package Installation::DiskActivation::DASDDiskManagementPage;
+use parent 'Installation::Navigation::NavigationBase';
+use strict;
+use warnings;
+
+sub init {
+    my ($self) = @_;
+    $self->SUPER::init();
+    $self->{tb_minimum_channel} = $self->{app}->textbox({id => 'min_chan'});
+    $self->{tb_maximum_channel} = $self->{app}->textbox({id => 'max_chan'});
+    $self->{btn_filter} = $self->{app}->button({id => 'filter'});
+    $self->{tbl_channels} = $self->{app}->table({id => 'table'});
+    $self->{menu_operation} = $self->{app}->menucollection({id => 'operation'});
+    return $self;
+}
+
+sub enter_minimum_channel {
+    my ($self, $minimum_channel) = @_;
```

Comment:
should be just `$channel`, when I read it I might think that there is some validation in place.